src/controllers/health.controllers.js

```diff
@@ -0,0 +1,10 @@
+export async function getStatusHandler(req, res, next) {
+	try {
+		res.status(200).json({ message: 'App is up and running', result: 'OK' });
+	} catch (error) {
+		if (!error.statusCode) {
```

Comment:
you already implement the same logic inside error handling middleware

Reply:
Done!